agent/heartbeat.go

```diff
@@ -8,12 +8,13 @@ import (
 // https://www.pivotaltracker.com/story/show/132265151
 
 type Heartbeat struct {
-	Deployment string            `json:"deployment"`
-	Job        *string           `json:"job"`
-	Index      *int              `json:"index"`
-	JobState   string            `json:"job_state"`
-	Vitals     boshvitals.Vitals `json:"vitals"`
-	NodeID     string            `json:"node_id"`
+	Deployment        string            `json:"deployment"`
+	Job               *string           `json:"job"`
+	Index             *int              `json:"index"`
+	JobState          string            `json:"job_state"`
+	Vitals            boshvitals.Vitals `json:"vitals"`
+	NodeID            string            `json:"node_id"`
+	NumberOfProcesses int               `json:"number_of_processes"`
```

Comment:
Maybe instead of `NumberOfProcesses` this becomes an embedded struct pointer (`ProcessStatus` ??) which can be `nil`, and possibly have more embedded fields that communicate `NumberOfProcess` and/or process-manager status...